clarifai/client/base.py

```diff
@@ -88,8 +88,12 @@ def from_auth_helper(cls, auth: ClarifaiAuthHelper, **kwargs):
       if kwargs.get("url"):
         default_kwargs.pop("user_id", "")
         default_kwargs.pop("app_id", "")
-      # Remove app_id if the class name contains "Runner"
-      if 'Runner' in _clss.__name__:
+      # Remove user_id if the class name contains "Nodepool"
+      # if 'Nodepool' in _clss.__name__:
+      #   default_kwargs.pop("user_id", "")
+      # Remove app_id if the class name contains "Runner", "ComputeCluster", "Nodepool", "Deployment"
+      co_resources = ['Runner', 'ComputeCluster', 'Nodepool', 'Deployment']
```

Comment:
How about moving this to a constants file?

Reply:
Done